models/Article.php

```diff
@@ -1,221 +1,157 @@
 <?php
-// Permet d'avoir accès à la base de données
-require_once "Model.php";
 
-// Cette classe sert à manipuler tout ce qui touche aux articles
-class Article extends Model
+class Article
 {
-// FONCTIONS POUR LE FRONTEND
-    
-    // Fonction qui permet de récupérer les détails des articles postés
-    public function get_posts_blog(){
-        $req = $this->db->query("
-            SELECT  articles.id,
-                    articles.title,
-                    articles.chapo,
-                    articles.image,
-                    articles.date,
-                    articles.content,
-                    admins.name
-            FROM articles
-            JOIN admins
-            ON articles.writer=admins.email
-            WHERE posted='1'
-            ORDER BY date DESC
-            LIMIT 0,5
-        ");
-        
-        $results = array();
-        
-        while($rows = $req->fetchObject()){
-            $results[] = $rows;
-        }
-        return $results;
+    private $id;
+    private $title;
+    private $chapo;
+    private $content;
+    private $date;
+    private $writer;
+    private $posted;
+    private $image;
+    
+    /**
+     * 
+     * @return mixed
+     */
+    public function getId()
+    {
+        return $this->id;
     }
     
-   public function get_posts_blog_all(){
-        $req = $this->db->query("
-            SELECT  articles.id,
-                    articles.title,
-                    articles.chapo,
-                    articles.image,
-                    articles.date,
-                    articles.content,
-                    admins.name
-            FROM articles
-            JOIN admins
-            ON articles.writer=admins.email
-            WHERE posted='1'
-            ORDER BY date DESC
-        ");
-        
-        $results = array();
-        
-        while($rows = $req->fetchObject()){
-            $results[] = $rows;
-        }
-        return $results;
+    /**
+     * 
+     * @param mixed $id
+     */
+    public function setId($id)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)